debs/bionic/fits/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: fits
+Section: utils
+Priority: extra
+Maintainer: austin trask <austin@artefactual.com>
+Build-Depends: debhelper (>= 7), default-jdk, ant-gcj
+Standards-Version: 3.8.3
+Homepage: http://code.google.com/p/fits/
+
+Package: fits
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, nailgun (>= 0.9), default-jdk
```

Comment:
A subtle problem here is that `default-jdk` in Bionic is going to install OpenJDK 11. So when we're installing MCPClient in Bionic we end up having both OpenJDK 8 and OpenJDK 11 installed. `bagit` and `jhove` depend on the former while `fits` depends on the latter.

```sh
$ dpkg-query --showformat='${Installed-Size}\t${Package}\t${Status}\n' -W "openjdk*" | grep "ok installed" | sort -k 1 -n  | awk '{printf "%.3f MB \t %s\n", $1/(1024), $2}'
0.167 MB 	 openjdk-11-jre
4.572 MB 	 openjdk-11-jdk
83.317 MB 	 openjdk-11-jdk-headless
95.348 MB 	 openjdk-8-jre-headless
174.862 MB 	 openjdk-11-jre-headless
```

That's 250+ MB that we could be saving if we could make it so only OpenJDK 8 is used across the board, like we do in Xenial. Would that be possible?

Reply:
FYI, related to `nailgun`:
- https://github.com/archivematica/Issues/issues/222
- https://github.com/archivematica/Issues/issues/223

Reply:
Updated  fits dependencies on bionic to depend on openjdk-8, and removed the references to nailgun-client